Update CLI sample deprovision script with parameters unification (#390)
box/developer.box.com: content/guides/cli/scripts/deprovision-users.md

## Configure the script

1. Clone the `boxcli` GitHub repository 
or download the files from [`examples`][examples] directory.

   ```bash
    git clone https://github.com/box/boxcli.git
   ```

2. Create the list of employees for deletion in `.csv`.

>    The header row should look like the below:

Reviewer: ```suggestion
   The header row should look like as follows:
```

>    `name,email`

Reviewer: I would use a codeblock:

```
name, email
```

   where:

   * `name` is the name of the user in Box. 
   * `email` is the primary email address of the user in Box.

   For example:
   
   |`name`| `email`|
   |------|--------|
   |Managed User 1| ManagedUser1@test.com|
   |Managed User 2| ManagedUser2@test.com|
   |Managed User 3| ManagedUser3@test.com|

> ### List of parameters

Reviewer: Let's create a table with parameters.

 
 ```bash
/Users_Deprovision.ps1 -SkipTransferContent
```

Otherwise user's content will be transferred.

`NewFilesOwnerID`: (Optional in interactive mode) The ID of the user to transfer files to before deleting the user.
If not specified, the script will prompt to input in the interactive mode, or use the current authenticated user ID to receive the content.

`EmployeeArchiveFolderName`: (Optional, default value: 'Employee Archive') The name of a folder, where users' content will be moved to if `SkipTransferContent` is set to `$false`.
If a folder with this name already exists in the user's `NewFilesOwnerID` root folder, it will be used. Otherwise, a new one will be created.

`DryRun`: (Optional, default value: `$false`) A flag that determines the script should be run in a mode, where no delete/create/update calls will be made, only read ones. The default is `$false`, which means that all calls will be made. To enable `dry run` mode, set this flat when running the script:

```bash
/Users_Deprovision.ps1 -DryRun
```

### Define script parameters

> There are 3 ways to pass parameters:

Reviewer: ```suggestion
You can use either of three ways to pass parameters.
```

> * Use hardcoded value in script:

Reviewer: ```suggestion
* Use hardcoded value in script.
```

> Please update all needed parameters in the script parameters section [here][parameters] before running.

Reviewer: ```suggestion
To use this option, update all required parameters listed in the [script parameters section][parameters] before running.
```

> * Run script with parameters:

Reviewer: ```suggestion
* Run script with parameters.
```

> You can also specify parameters while run the command, for example:

Reviewer: ```suggestion
You can also specify parameters while providing the command. For example:
```

```bash
PS > ./Users_Deprovision.ps1 -EmployeeList ./Employees_to_delete.csv `
   -NewFilesOwnerID  123456789`
   -EmployeeArchiveFolderName "Employee Archive"

Starting User Deprovisioning script...
```

> * If you don't specify parameters, the script will prompt you to enter it.

Reviewer: ```suggestion
If you don't specify parameters, the script will prompt you to enter it.
```
I would skip the bullet point here :)
